app/models/case/base.rb

```diff
@@ -660,6 +660,10 @@ def current_team_and_user_resolver
     CurrentTeamAndUser::Base.new(self)
   end
 
+  def extended_for_pit?
+    currently_extended_for_pit?
+  end
```

Comment:
shouldn't currently_extended_for_pit? be renamed as extended_for_pit? and replace this method?